include/z64camera.h

```diff
@@ -835,56 +841,72 @@ typedef struct {
     { swingPitchAdj, CAM_DATA_SWING_PITCH_ADJ }, \
     { fov, CAM_DATA_FOV }, \
     { atLerpStepScale, CAM_DATA_AT_LERP_STEP_SCALE }, \
-    { yawUpdateRateTarget, CAM_DATA_YAW_UPDATE_RATE_TARGET }, \
+    { initTimer, CAM_DATA_YAW_UPDATE_RATE_TARGET }, \
```

Comment:
Does the second value (CAM_DATA_YAW_UPDATE_RATE_TARGET) also need to be updated?